inc/authldap.class.php

```diff
@@ -3945,4 +4003,31 @@ public static function getUsers($values, &$results, &$limitexceeded) {
 
       return $users;
    }
+
+   public function checkFilesExist(&$input) {
+
+      if (isset($input['tls_certfile'])) {
```

Comment:
I guess we should allow removal of field value.
```suggestion
      if (isset($input['tls_certfile']) && !empty($input['tls_certfile'])) {
```